lib/ActivityStreams.js

```diff
@@ -592,12 +592,10 @@ ActivityStreams.prototype = {
     };
 
     switch (reason){
-      // can be one of: disable/shutdown/upgrade/downgrade
-
-      // note that if the user uninstalls the addon, the reason will be set as
-      // "disable", see more here https://developer.mozilla.org/en-US/Add-ons/SDK/Tutorials/Listening_for_load_and_unload
+      // can be one of: uninstall/disable/shutdown/upgrade/downgrade
       case "disable":
-        this._tabTracker.handleUserEvent({event: "disable"});
+      case "uninstall":
+        this._tabTracker.handleUserEvent({event: reason});
         this._previewProvider.clearCache();
         this._unsetHomePage();
```

Comment:
this is called for both disable and uninstall, isn't it? /me reads up on js switch statements.